meshroom/ui/qml/ImageGallery/ImageGallery.qml

```diff
@@ -438,7 +441,16 @@ Panel {
                 anchors.fill: parent
                 enabled: !m.readOnly && !intrinsicsFilterButton.checked
                 keys: ["text/uri-list"]
-                // TODO: onEntered: call specific method to filter files based on extension
+                onEntered: {
+                    isMeshroomScene = false
+                    nbFilesDropped = drag.urls.length
```

Comment:
Not a very accurate variable name since no file has been dropped yet. As we're still in the "dragging" phase, `nbDraggedFiles` might be more explicit.